### ISSUE TITLE
Regarding images in the Web Folder

### ISSUE THREAD

**Issue by ArjunPukale** [junyanz/pytorch-CycleGAN-and-pix2pix#1253]
During training the images which are displayed in the web folder are training set images or from the test set?

**Comment by improgress** [comment]
of course the traindataset

**Comment by junyanz** [comment]
They are training set images. You need to run `test.py` for test image results.
